web/api/v1/api.go

```diff
@@ -115,11 +129,11 @@ type API struct {
 
 	targetRetriever       targetRetriever
 	alertmanagerRetriever alertmanagerRetriever
-
-	now      func() time.Time
-	config   func() config.Config
-	flagsMap map[string]string
-	ready    func(http.HandlerFunc) http.HandlerFunc
+	alertsrulesRetreiver  alertsrulesRetreiver
```

Comment:
retriever

Reply:
if  you referring to the spelling of retreiver i.e. it is mis-spelt .. i will fix it